soccer/gameplay/plays/tuning/pid.py

```diff
@@ -0,0 +1,80 @@
+import behavior
+import robocup
+import main
+import enum
+import constants
+import tactics.line_up
+import tactics.tune_pid
+import skills.move
+import play
+
+
+# plan is for this to eventually loop through all robots and tune one at a time
+class pid(play.Play):
+    class State(enum.Enum):
+        prep = 1  #go to the start point, line up other bots
+        testing = 2  # Drive a test line
+
+    def __init__(self):
+        super().__init__(continuous=True)
+
+        for state in pid.State:
+            self.add_state(state, behavior.Behavior.State.running)
+
+        self.add_transition(behavior.Behavior.State.start, pid.State.prep,
+                            lambda: True, 'immediately')
+
+        self.add_transition(
+            pid.State.prep, pid.State.testing,
+            lambda: self.has_subbehavior_with_name('move2') and self.subbehavior_with_name('move2').state == behavior.Behavior.State.completed and self.subbehavior_with_name('move2').robot.vel.mag() < .05,
```

Comment:
Is there a way to format this so the line isn't as long? Either split it up into multiple lines or into a function?